### ISSUE TITLE
More detial about mobilenetv3 train.

### ISSUE THREAD

**Issue by myzhuang** [ztsrxh/RSCD-Road_Surface_Classification_Dataset#2]
Thanks for your share！And can you provide more detail about training setting of MobileNetV3, such as train epoch, learning rate, and input image resloution?

**Comment by ztsrxh** [comment]
Hello, thanks for your attention to our work. We reviewed the codes and found some problems with the result and the newest results are updated now. 
The batch-size is 128, learning-rate is 0.001 for Adam, input image size is just 360*240.
